src/features/recovery/components/RecoveryContext/RecoveryContextHooks.tsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react'
+import { useRecoveryState } from './useRecoveryState'
+import { useRecoveryDelayModifiers } from './useRecoveryDelayModifiers'
+import { useRecoveryPendingTxs } from './useRecoveryPendingTxs'
+import { useRecoverySuccessEvents } from './useRecoverySuccessEvents'
+import store from '.'
+
+function RecoveryContextHooks(): null {
+  const [delayModifiers, delayModifiersError, delayModifiersLoading] = useRecoveryDelayModifiers()
+  const [recoveryState, recoveryStateError, recoveryStateLoading] = useRecoveryState(delayModifiers)
+  const pending = useRecoveryPendingTxs()
+
+  useRecoverySuccessEvents(pending, recoveryState)
```

Comment:
This hook is apparently used for side effects and it shouldn't be in a store intializer...